base:
    # 'j2s7s300' is the name of the 7DOF gen2 kinova jaco arm
    # 'j2s6s300' is the name of the 6DOF gen2 kinova jaco arm 
    name: 'j2n6s300' 
    # port to use for interface to receive step/reset commands from
    server_port: 10001
    # only velocity controller is implemented
    active_controller: 'velocity'
# from https://github.com/sahandrez/jaco_control/blob/master/cfg/controller_gains.cfg
velocity_kp_gains:
    joint_1: [5.0, 0.0, 200.]
    joint_2: [5.0, 0.0, 200.]
    joint_3: [5.0, 0.0, 200.]
    joint_4: [5.0, 0.0, 200.]
    joint_5: [5.0, 0.0, 200.]
    joint_6: [5.0, 0.0, 200.]
velocity_kd_gains:
    joint_1: [1.0, 0.0, 50.]
    joint_2: [1.0, 0.0, 50.]
    joint_3: [1.0, 0.0, 50.]
    joint_4: [1.0, 0.0, 50.]
    joint_5: [1.0, 0.0, 50.]
    joint_6: [1.0, 0.0, 50.]
    

